pycqed/instrument_drivers/meta_instrument/qubit_objects/qubit_object.py

```diff
@@ -103,7 +103,8 @@ def measure_spectroscopy(self):
     def measure_transients(self):
         raise NotImplementedError()
 
-    def calibrate_optimal_weights(self):
+    def calibrate_optimal_weights(self, MC=None, verify=True,
```

Comment:
I would prefer to have a more general 'set_weights( )' function. Based on three qubit object parameters(all booleans) a different calibration/setting is performed:
- SSB: determins whether we use single sideband or double sideband measurement
- optimized weights: determines whether weight functions are measured or standard weight functions are set 
- one weight functions, only loads weight functions into the I channel